pytapo/__init__.py

```diff
@@ -1086,6 +1086,80 @@ def getWhitelampStatus(self):
             "getWhitelampStatus", {"image": {"get_wtl_status": ["null"]}}
         )
 
+    ### TODO: UNTESTED
+    def getFloodlightStatus(self):
+        return self.executeFunction(
+            "getFloodlightStatus", {"floodlight": {"get_floodlight_status": ""}}
+        )
+
+    ### TODO: UNTESTED
+    def setFloodlightStatus(self, status):
+        return self.executeFunction(
+            "setFloodlightStatus",
+            {"floodlight": {"set_floodlight_status": 1 if status else 0}},
+        )
+
+    ### TODO: UNTESTED
+    def getFloodlightConfig(self):
+        return self.executeFunction(
+            "getFloodlightConfig", {"floodlight": {"name": "config"}}
+        )
```

Comment:
If it works, we need to probably extract some specific property, just like in other functions